onto_spread_ed/config.py

```diff
@@ -90,13 +92,17 @@
     REPOSITORIES = {
         "BCIO": "b-gehrke/ontologies",
         "AddictO": "b-gehrke/addiction-ontology",
-        "GMHO": "b-gehrke/mental-health-ontology"
+        "GMHO": "b-gehrke/mental-health-ontology",
```

Comment:
I think, we should keep this section and the following as it is.

Reply:
Yeah Björn, this is as it is. We have just added our names as well.

Reply:
Right, but it adds errors as the key `BCIO` is defined multiple times. See the failed check. In the future, it would probably be a good idea to have these user specific settings in another file but I'm not sure yet how to solve this properly. For now, only one entry should be committed.